libobs/obs-source.c

```diff
@@ -1506,11 +1506,11 @@ static void source_output_audio_data(obs_source_t *source,
 			push_back = true;
 
 			/* This typically only happens if used with async video when
-		 * audio/video start transitioning in to a timestamp jump.
-		 * Audio will typically have a timestamp jump, and then video
-		 * will have a timestamp jump.  If that case is encountered,
-		 * just clear the audio data in that small window and force a
-		 * resync.  This handles all cases rather than just looping. */
+			 * audio/video start transitioning in to a timestamp jump.
+			 * Audio will typically have a timestamp jump, and then video
+			 * will have a timestamp jump.  If that case is encountered,
+			 * just clear the audio data in that small window and force a
+			 * resync.  This handles all cases rather than just looping. */
```

Comment:
This actually looks like it's a comment about the below `else if`, in which case the first line should probably be de-indented. That, or the whole comment block gets moved into the `else if` so that the indentation makes sense.

Reply:
The comment is moved inside the closure for `if (diff > MAX_TS_VAR)`.